ci-operator/step-registry/openshift/image-based/infra/sos/openshift-image-based-infra-sos-commands.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -xeuo pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+IP_ADDRESS="$(cat "${SHARED_DIR}/public_address")"
+HOST_USER="$(cat "${SHARED_DIR}/ssh_user")"
+INSTANCE_PREFIX="${HOST_USER}@${IP_ADDRESS}"
+
+echo "Using Host $IP_ADDRESS"
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=ERROR
+  -i "${CLUSTER_PROFILE_DIR}/ssh-privatekey")
+
+ssh "${SSHOPTS[@]}" "${INSTANCE_PREFIX}" <<'EOF'
+  set -x
+  if ! hash sos ; then
+    sudo touch /tmp/sosreport-command-does-not-exist
+    exit 0
+  fi
+
+  plugin_list="container,network"
+  if ! sudo sos report --list-plugins | grep 'microshift.*inactive' ; then
+    plugin_list+=",microshift"
+  fi
```

Comment:
I think I recall Jeff had intended to keep things similar to MicroShift to merge some of these files later. But I'll remove it to avoid any confusion for now.